src/app-layout/utils/use-keyboard-events.ts

```diff
@@ -21,13 +22,17 @@ const getCurrentSize = (panelRef?: React.RefObject<HTMLDivElement>) => {
   };
 };
 
-export const useKeyboardEvents = ({ position, onResize, panelRef }: SizeControlProps) => {
+export const useKeyboardEvents = ({ position, onResize, panelRef, hasTransitions = false }: SizeControlProps) => {
   return (event: React.KeyboardEvent) => {
     let currentSize;
     let maxSize;
 
     const { panelHeight, panelWidth } = getCurrentSize(panelRef);
 
+    if (hasTransitions) {
+      panelRef?.current?.classList.remove(styles['with-motion']);
```

Comment:
We do not add this class programmatically, React does it in `<ActiveDrawer />`. Removing it imperatively may put it out of think with React, which may think the class is still there.

It should always be symmetric. Either we fully manage the class via `classList.add` and `classList.remove` or we fully delegate it to React via `useState` or something.

I am fine either way as long as they do not mix